x-pack/plugins/observability_solution/apm/public/components/app/entities/entity_link/index.tsx

```diff
@@ -0,0 +1,132 @@
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiEmptyPrompt, EuiLink, EuiLoadingSpinner, EuiImage } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import { dashboardsDark, dashboardsLight } from '@kbn/shared-svg';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { ENVIRONMENT_ALL_VALUE } from '../../../../../common/environment_filter_values';
+import { useServiceEntitySummaryFetcher } from '../../../../context/apm_service/use_service_entity_summary_fetcher';
+import { useEntityManagerEnablementContext } from '../../../../context/entity_manager_context/use_entity_manager_enablement_context';
+import { useApmParams } from '../../../../hooks/use_apm_params';
+import { isPending } from '../../../../hooks/use_fetcher';
+import { useApmRouter } from '../../../../hooks/use_apm_router';
+import { useTheme } from '../../../../hooks/use_theme';
+import { ApmPluginStartDeps } from '../../../../plugin';
+
+const pageHeader = {
+  pageTitle: 'APM',
+};
+
+export function EntityLink() {
+  const router = useApmRouter({ prependBasePath: false });
+  const theme = useTheme();
+  const { services } = useKibana<ApmPluginStartDeps>();
+  const { observabilityShared, data } = services;
+  const timeRange = data.query.timefilter.timefilter.getTime();
+  const ObservabilityPageTemplate = observabilityShared.navigation.PageTemplate;
+  const {
+    path: { serviceName },
+    query: { rangeFrom = timeRange.from, rangeTo = timeRange.to },
+  } = useApmParams('/link-to/entity/{serviceName}');
+  const { isEntityCentricExperienceViewEnabled, entityManagerEnablementStatus } =
+    useEntityManagerEnablementContext();
+  const { serviceEntitySummary, serviceEntitySummaryStatus } = useServiceEntitySummaryFetcher({
+    serviceName,
+    environment: ENVIRONMENT_ALL_VALUE,
+  });
+
+  if (isPending(entityManagerEnablementStatus) || isPending(serviceEntitySummaryStatus)) {
+    return (
+      <div>
+        <EuiLoadingSpinner />
+      </div>
+    );
+  }
+
+  if (
+    // When EEM is disabled we'll show the APM overview page.
+    isEntityCentricExperienceViewEnabled === false ||
+    // When EEM is enabled and the Service has APM and/or Logs data, we'll show the APM overview page
+    (serviceEntitySummary?.dataStreamTypes && serviceEntitySummary.dataStreamTypes.length > 0)
+  ) {
+    return (
+      <Redirect
+        to={router.link('/services/{serviceName}/overview', {
+          path: { serviceName },
+          query: {
+            rangeFrom,
+            rangeTo,
+            kuery: '',
+            serviceGroup: '',
+            comparisonEnabled: true,
+            environment: ENVIRONMENT_ALL_VALUE,
+          },
+        })}
+      />
+    );
+  }
+
+  // When EEM is enabled and the service is not found on the EEM indices display a callout guiding on the limitations of EEM
```

Comment:
Is it possible that this callout isn’t visible in the videos you shared in the PR description? It might be helpful to add a screenshot of it.